.github/workflows/setupmc.yml

```diff
@@ -0,0 +1,27 @@
+            - name: Setup Minecraft
+  # You may pin to the exact commit or the version.
+  # uses: mcenv/setup-minecraft@2d092242881552b8ee463d9b4687bd38ef58a396
+  uses: mcenv/setup-minecraft@v3.3.2
+  with:
+    # Minecraft version to use
+    version: 1.12.2
+    # Whether to install Minecraft
+    install: # optional, default is true
+    # Whether to cache Minecraft
+    cache: # optional
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (performance):** Undefined 'cache' option.

The 'cache' option is mentioned as optional but lacks a value or a default indication. Defining whether caching is enabled or not could improve build performance.

```suggestion
    cache: true # optional, default is true
```